lfs/batcher.go

```diff
@@ -8,21 +8,41 @@ package lfs
 //   * Exit() is called
 // When a timeout, Flush(), or Exit() occurs, the group may be smaller than the
 // batch size.
+
+// A Lot represents a group of Transferables that was packaged up by a Batcher.
+type Lot []Transferable
```

Comment:
Batch was my first choice, but is already defined in this package [here](https://github.com/github/git-lfs/blob/master/lfs/client.go#L177). I'm not to averse of `Lot`, but `TransferableGroup` would also work well imho.